src/sql-editor/standardSql/types.ts

```diff
@@ -41,4 +48,7 @@ export interface SQLMonarchLanguage extends monacoTypes.languages.IMonarchLangua
   builtinFunctions?: string[];
   logicalOperators?: string[];
   comparisonOperators?: string[];
+
+  //used by basic languages in the monaco registry
```

Comment:
total nit, but I think if we write the comments like this:
```
  /** used by basic languages in the monaco registry */
  operators?: string[];

```

it'll show up like this in vscode:
<img width="521" alt="Screen Shot 2022-07-25 at 3 07 29 PM" src="https://user-images.githubusercontent.com/6620164/180855356-0aeb235f-59cb-4226-b401-8c4345bb6c0b.png">

Reply:
also now that I think about it I'm not sure I know what this comment means, What is an operator vs a logical/comparison operator?

Reply:
Operator is _all_ operators - `<>`, `AND` etc. By separating logical operators from comparison operators, we can provide better autocomplete since they should be suggested in different contexts. Added a new comment. Hope it's a little clearer now!